### ISSUE TITLE
Meteor Pages doesn't recognize the collection helpers

### ISSUE THREAD

**Issue by MaazAli** [alethes/meteor-pages#66]
I've been using this plugin for displaying some of my collections. I use `collection-helpers` to add additional attributes to collections. However, Meteor-Pages doesn't recognize the helpers, while everything else in Meteor does. 

Here is the code that I am currently using. I had to make another template specifically for this plugin, because collection helpers aren't working. Here is the plugin that I am talking about https://github.com/dburles/meteor-collection-helpers

And this is what the author of the plugin had to say about this: https://github.com/dburles/meteor-collection-helpers/issues/11

```
AnimePages = new Meteor.Pagination(Anime, {
    router: 'iron-router',
    routerTemplate: 'animeExplore',
    homeRoute: '/anime/explore/',
    route: '/anime/explore/page/',
    perPage: 30,
    itemTemplate: 'animeCardProxy',
    routerLayout: 'defaultLayout',
    sort: {canonicalTitle: 1},
    templateName: 'animeSpecificExplore',
    /*infiniteItemsLimit: 30,*/

});
```

```
<template name="animeExplore">

    {{> animeSpecificExplore}}

</template>

<template name="animeSpecificExplore">

<div class="container main-content">

    {{> pagesNav}}

    <div class="row">
        {{> pages}}
    </div>

    {{> pagesNav}}

</div>

</template>
```

**Comment by ribbedcrown** [comment]
@MaazAli, I have the same issue too, are you able to solve this problem?

**Comment by aadamsx** [comment]
I think this project is no longer maintained.  I ended up doing a list or table with a load more button at the bottom -- works great.

**Comment by ribbedcrown** [comment]
@aadamsx Clearly, thanks for the reply!

**Comment by kalarani** [comment]
I'm running into the same issue too. Is there another pagination package that any of you would recommend?

**Comment by dsjoerg** [comment]
+1 on @kalarani's question

**Comment by sjmcdowall** [comment]
https://github.com/alethes/meteor-pages/issues/184

Have you read that issue and followed up with the other package named?

**Comment by dsjoerg** [comment]
@sjmcdowall yes.  However my primary reason for asking is to see if the Meteor community is aware of alternative packages for pagination, that is, alternatives to this project and its fork.  Can't complain about free software, but I'm hoping there's something out there that's both up-to-date and has working non-trivial examples, in particular examples that use auth() returning a cursor.

**Comment by sjmcdowall** [comment]
Ah -- yes -- pagination (server side) seems to be a hot topic these days.  I looked at Atmosphere (I presume you have as well) and really didn't find anything either that was what I wanted to so I rolled by own solution specific to my needs too.  And yes, the issue appears to be that all examples are too trivial for real world work (unless you have a trivial app I guess).  In my guess I had multiple sorts and multiple different filtering options, etc.

**Comment by aadamsx** [comment]
Same here, had to roll my own solution to get things work right.

**Comment by yasaricli** [comment]
+1
